packages/ui/src/proposals/components/StatisticsPreview/ProposalStatistics.tsx

```diff
@@ -31,9 +31,10 @@ export const ProposalStatistics = ({ voteCount, constants }: ProposalStatisticsP
   const approvalQuorum = councilSize && approvalQuorumRatio && Math.ceil(councilSize * approvalQuorumRatio)
   const slashingQuorum = councilSize && slashingQuorumRatio && Math.ceil(councilSize * slashingQuorumRatio)
 
-  const quorumRatio = councilSize ? total / councilSize : 0
-  const approvalRatio = total && approve / total
-  const slashingRatio = total && slash / total
+  const approvalCurrentRatio = councilSize ? (total - abstain) / councilSize : 0
+  const slashingCurrentRatio = councilSize ? (total - abstain) / councilSize : 0
```

Comment:
Please use one variable just to make it clear that it's one value:
```suggestion
  const quorumRatio = councilSize ? (total - abstain) / councilSize : 0
```